api/src/main/resources/workbench.yaml

```diff
@@ -638,6 +642,46 @@ paths:
           schema:
             $ref: "#/definitions/WorkspaceListResponse"
 
+  /v1/users/id-verification/list:
```

Comment:
Since you've kind of started a convention of calling things "admin" when they need more permission, it would be good to expose that through the URL. I suggest, `/v1/admin/users/id-verification/list`.

Reply:
Agreed.